modules/Individual Needs/in_view.php

```diff
@@ -37,8 +37,9 @@
 
         $studentGateway = $container->get(StudentGateway::class);
 
-        $gibbonPersonID = isset($_GET['gibbonPersonID'])? $_GET['gibbonPersonID'] : '';
-        $search = isset($_GET['search'])? $_GET['search'] : '';
+        $gibbonPersonID = $_GET['gibbonPersonID'] ?? '';
+        $search = $_GET['search'] ?? '';
+        //Why is this in brackets?
         $allStudents = (isset($_GET['allStudents']) ? $_GET['allStudents'] : '');
```

Comment:
I wonder how you might answer that question yourself. It's good to ask questions, but in this case you've skipped this line rather than attempting to refactor it and seeing the result.